tests/common.py

```diff
@@ -21,14 +21,19 @@
 def _reset_schema():
     rest.db.engine.execute('drop schema if exists public cascade;')
     rest.db.engine.execute('drop schema if exists disclosure cascade;')
+    rest.db.engine.execute('drop schema if exists staging cascade;')
     rest.db.engine.execute('create schema public;')
     rest.db.engine.execute('create schema disclosure;')
+    rest.db.engine.execute('create schema staging;')
```

Comment:
This has to with the same problem I was having in the PR for migrating some tables to the `disclosure` schema.  The integration tests fail if those create schema statements are there. And the reason, or so I suspect, is in the subset dump the dump is exporting a statement to create those other schemas (but not the public schema!).  So having the create statements in the test logic was clashing and causing an error in the tests.  I did a little research to see if there is a way to exclude those create schema statements but didn't find anything.  Great documentation for leaving out tables; just not schemas.  Maybe can create an issue to get to later under `py.test grooming` from our quarterly planning.